### PR TITLE
Error line pallete bug fix

### PR DESCRIPTION
**Pull request by CMGreenspon**
Explicitly counts number of primary series as the length of the series object no longer seems to track that

### CODE REVIEW

**Comment by CMGreenspon** [comment]
Closes #523

File: src/errorline.jl [hunk 1]
```diff
@@ -50,45 +50,45 @@ errorline(1:10, y)
...
+    if errortype === :std || errortype === :sem
+        y_error = mapslices(NaNMath.std, y, dims=2)
+        if errortype == :sem
+            y_error = y_error ./ sqrt(size(y,2))
         end
 
-    elseif errortype === :percentile
-        y_lower = fill(NaN, size(y, 1))
-        y_upper = fill(NaN, size(y, 1))
+    elseif errortype == :percentile
```
Comment (BeastyBlacksmith) [inline src/errorline.jl:80]: would be good to consistently use `===` for comparisons with symbols

**Comment by BeastyBlacksmith** [comment]
Can you add a test that checks that you get the correct color for the example in #523 ?

**Comment by CMGreenspon** [comment]
I can add the check but I have a bunch of deadlines before I go on holiday for a week so it will have to wait until the week of the 20th.

**Pull request merged**